include/robot_localization/ros_filter.h

```diff
@@ -165,10 +166,11 @@ template<class T> class RosFilter
                    std::vector<int> &updateVector);
 
     //! @brief Retrieves the EKF's output for broadcasting
-    //! @param[out] message - The standard ROS odometry message to be filled
+    //! @param[out] odomMsg - The standard ROS odometry message to be filled
+    //! @param[out] accelMsg - The standard ROS acceleration message to be filled
     //! @return true if the filter is initialized, false otherwise
     //!
-    bool getFilteredOdometryMessage(nav_msgs::Odometry &message);
+    bool getFilteredOdometryAndAccelMessage(nav_msgs::Odometry &odomMsg, geometry_msgs::AccelWithCovarianceStamped& accelMsg);
```

Comment:
I think I'd prefer that we add a second method for assembling the acceleration message. Then we can fill it conditionally base on the value of `publishAcceleration_`, and we're not changing this method signature.